index.html

```diff
@@ -72,12 +72,12 @@
       <div class="cta-section">
         <a href='pages/stable-version.html' class="btn btn-primary">
           <span class="badge eos-icons">get_app</span>
-          Get new 2020.05!
+          Download latest
```

Comment:
Let's keep the old text.

Reason: the news are not on top, and at least people can notice what the latest version is.

Reply:
OK